crates/wallet/src/wallet/params.rs

```diff
@@ -161,13 +161,13 @@ impl LoadParams {
     }
 
     /// Check for `network`.
-    pub fn network(mut self, network: Network) -> Self {
+    pub fn validate_network(mut self, network: Network) -> Self {
```

Comment:
⛏️ A small nit, but what do you think about calling these functions `check_network` and `check_genesis_hash`  to match internal field names?  it's a little shorter/easier to type with the same meaning.

Reply:
Yep. As mentioned above I wasn't set between the two and I'm happy to use `check` rather than `validate` if that's preferred. Fixing now!